### DISCUSSION TITLE
Wildcard triggers for flows?

### DISCUSSION THREAD

**Discussion by m1212e** [directus/directus#16008] [Archive]
Is there a way to allow wildcard triggers in flows?

I'd like to execute a flow whenever a non system collection is updated, no matter which one. When I add new collections, I don't want to add them manually to the flow.

**Comment by nickrum** [comment]
I don't think this is currently possible.

One option would be to trigger flows with `event` trigger and `items.create`, `items.update` or `items.delete` scope on all collections if no collections are specified. That's more or less how the events in hook extensions work. The problem with this approach is that, while hooks have dedicated events for system collections, flows use the same collection selection interface to trigger on system collections. So it might be confusing if we would only trigger on non-system collections if no collections are specified.

Alternatively, we could add a special key indicating that we want to trigger on non-system collections.

**Reply by m1212e** [reply]
That would be really helpful!
